src/overlays/actors/ovl_Obj_Switch/z_obj_switch.h

```diff
@@ -58,7 +58,7 @@ typedef struct ObjSwitch {
     /* 0x017A */ u8 x2TexScroll;
     /* 0x017B */ u8 y2TexScroll;
     /* 0x017C */ Color_RGB8 crystalColor;
-    /* 0x017F */ u8 unk_17F; // used for different purposes between floor and eye switch
+    /* 0x017F */ u8 unk_17F; // seems to be used to check if a switch has been hit by an AC collider
```

Comment:
AC is Attack Checker so something's wrong here
I would probably just get rid of the comment entirely tbh

Reply:
From what I understand this is used to detect when the player wants to trigger the switch, hence the AC check

Reply:
right, but you phrased your comment as "hit by an AC collider"
and AC collider (blue in gz) gets hit by AT colliders (red), so its not the AC collider doing the hitting

at a quick glance its also used for more than that though, it stores the value of a dynapoly variable we havent named. 
agreed on just removing the comment, it will get a proper name when we can name it

Reply:
oh ok, I'll remove it then